deployments/charts/kuma/README.md

```diff
@@ -13,6 +13,7 @@ A Helm chart for the Kuma Control Plane
 | global.image.registry | string | `"kong-docker-kuma-docker.bintray.io"` | Default registry for all Kuma Images |
 | global.image.tag | string | `nil` | The default tag for all Kuma images, which itself defaults to .Chart.AppVersion |
 | patchSystemNamespace | bool | `true` | Whether or not to patch the target namespace with the system label |
+| installNewCrdsWhenUpgrade | bool | `true` | Whether ot not install new CRDs before upgrade  (if any were introduced    with the new version of Kuma) |
```

Comment:
What's the advisable usage of this flag? Should users skip installing CRDs with Helm? Would it make sense to only use this method of installing CRDs, and use a pre-install hook as well?

Reply:
So our main goal for this flag is to be used during the `helm upgrade` process inside the `pre-upgrade` hook, as helm is not installing the new crds during the upgrade process by itself and it had bitten us already few times as many people working with helm is not aware of this

ref:
- https://helm.sh/docs/topics/charts/#limitations-on-crds
- https://github.com/helm/community/blob/f9e06c16d89ccea1bea77c01a6a96ae3b309f823/architecture/crds.md

Reply:
Sounds great to me, thanks!